app/models/simulation.rb

```diff
@@ -7,4 +7,7 @@ class Simulation < ActiveRecord::Base
   has_many :content, class_name: 'Content'
 
   accepts_nested_attributes_for :users
+
+  has_one :name
+  has_one :requester
```

Comment:
Wow I was definitely having a brain fart when I wrote this. You're right, we don't need models for either of them. I don't think we need a user model unless we add logging in. I think this would make more sense:
```
attr_accessible :name
attr_accessible :requester
```
And modify schema:
```
  create_table "simulations", force: :cascade do |t|
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "name"
+    t.string "requester"
  end